.gitignore

```diff
@@ -24,7 +24,7 @@ fileHashes.lock
 functional-output/xui
 test/functional/a11y_output.js
 test/functional/a11y.html
-e2e-aat.env
+*e2e-aat.env
```

Comment:
Can we just update line 39 to be *.env, I'm going to switch the readme out on my next functional test PR to just use .env file.